plugin/adsk/plugin/plugin.cpp

```diff
@@ -101,52 +87,6 @@ const MTypeId MayaUsdPreviewSurface_typeId(0x58000096);
 const MString MayaUsdPreviewSurface_typeName("usdPreviewSurface");
 const MString MayaUsdPreviewSurface_registrantId("mayaUsdPlugin");
 
-#if defined(WANT_UFE_BUILD) && defined(UFE_V2_FEATURES_AVAILABLE)
-// Keep a reference to the existing USD Transform3d handler, to restore on
-// finalization.
-Ufe::Transform3dHandler::Ptr g_OldTransform3dHandler;
-Ufe::Transform3dHandler::Ptr g_NewTransform3dHandler;
-
-class ToggleTransform3d : public MPxCommand
```

Comment:
This command was a hack used to revert to Maya 2020 behavior. We don't support this anymore. That means we no longer need the UsdTransform3dHandler and UsdTransform3d.